ownership/sync-repos.py

```diff
@@ -0,0 +1,255 @@
+"""
+Synchronize ownership information between repositories and spreadsheet.
+
+Finds mismatches between the repositories listed in the ownership
+tracking spreadsheet and the actual list of repositories in a
+collection of Github organizations.
+"""
+
+import click
+import csv
+import github3
+from github3.exceptions import NotFoundError
+from itertools import groupby
+import re
+
+def bail(message):
+    raise KnownError(message)
```

Comment:
I can probably get rid of `bail` as a layer of indirection. I think it was originally going to be a little fancier.

Reply:
Agree, let's get rid of this indirection.